livre/core/data/MemoryUnit.h

```diff
@@ -81,15 +75,13 @@ class NoMemoryUnit : public MemoryUnit
 class ConstMemoryUnit : public MemoryUnit
 {
 public:
-    ConstMemoryUnit( const uint8_t* ptr, const size_t size );
+    explicit ConstMemoryUnit( const uint8_t* ptr );
```

Comment:
How is this possible that the size is unused? With a raw pointer you always need a size, the only exception I know of is the '\0' terminated c_str, but no such convention seems to be used / documented here...